integration_tests/suite/status/test_confd_status.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
```

Comment:
Can you move this file to `integration_tests/suite/base/test_status.py`

Reply:
Done :)